odoo/orm/fields.py

```diff
@@ -1052,6 +1052,8 @@ def convert_to_export(self, value, record):
         """ Convert ``value`` from the record format to the export format. """
         if not value:
             return ''
+        elif type(value).__name__ == "Property" and type(value).__module__ == "odoo.orm.fields_properties":
+            return value._values
```

Comment:
```suggestion
```

You can't apply this fix here.

You should apply the fix at the Properties class instead at below file.
https://github.com/odoo/odoo/blob/e61d55c6bf8afaef37d813a702438cb96baeddb1/odoo/orm/fields_properties.py#L209

```py
def convert_to_export(self, value, record):
    """ Convert ``value`` from the record format to the export format."""
    return self.convert_to_cache(value, record)
```